Makefile

```diff
@@ -597,9 +597,7 @@ tele-mac: flags
 # goinstall builds and installs gravity locally
 #
 .PHONY: goinstall
-goinstall: remove-temp-files compile
-	mkdir -p $(GRAVITY_BUILDDIR)
-	mkdir -p $(TF_PROVIDER_DIR)
+goinstall: remove-temp-files compile | $(TF_PROVIDER_DIR) $(GRAVITY_BUILDDIR)
```

Comment:
This is slick!  I haven't seen order-only prerequistes used before.  But reading through https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html makes me appreciate how they're used with directories.  I definitely could use this in some of my Make work.